operators/operating-pinot/tuning/routing.md

```diff
@@ -97,4 +115,57 @@ column.memberId.numPartitions = 4
 column.memberId.partitionValues = 1
 ```
 
-Broker side pruning for partitioning can be configured by setting the `segmentPrunerTypes` in the `RoutingConfig`. Note that the current implementation for partitioning only works for **EQUALITY** and **IN** filter (e.g. `memberId = xx`, `memberId IN (x, y, z)`).
+It is important to note that Pinot uses the partition configuration as metainformation.
+On offline tables, Pinot assumes that segments actually honor the specified partition, so the ingestion process must guarantee that.
```

Comment:
Can we add a bit more information on what needs to be guaranteed? When ingesting offline data, we usually expect that each input file contains all the relevant rows for a single partition.

Reply:
> we usually expect that each input file contains all the relevant rows for a single partition.

Is that correct? I thought what we expect is that all cells in the column that is partitioned belong to the same partition. But a single segment doesn't have to contain all rows of a single partition.

Reply:
I've added an example (although it is almost a repetition from the one we have above)